src/OrchardCore.Build/Dependencies.props

```diff
@@ -9,6 +9,7 @@
 
   <ItemGroup>
     <PackageManagement Include="Azure.Storage.Blobs" Version="12.4.0" />
+    <PackageManagement Include="BenchmarkDotNet" Version="0.12.1" />
```

Comment:
I'm not sure if the intend or `Dependencies.props` to be used in source code or can includes a unit tests projects